src/OpenTelemetry.Exporter.Zipkin/README.md

```diff
@@ -15,12 +15,14 @@ dotnet add package OpenTelemetry.Exporter.Zipkin
 
 ## Configuration
 
-You can configure the `ZipkinExporter` by following the directions below:
+You can configure the `ZipkinExporter` with the following options:
 
-* `Endpoint`: Zipkin endpoint address.
-* `TimeoutSeconds`: Timeout in seconds.
 * `ServiceName`: Name of the service reporting telemetry.
-* `UseShortTraceIds`: Value indicating whether short trace id should be used.
+* `Endpoint`: URI address to receive telemetry.
+* `UseShortTraceIds`: Whether the trace's ID should be shortened before
+   sending to Zipkin (default false).
+* `MaxPayloadSizeInBytes`: Maximum payload size - for .NET versions below
```

Comment:
for .NET versions **other** than .NET 4.5.2.

Reply:
Thanks for the catch, it's been fixed in commits. Can I ask what is special about .NET 4.5.2 that this parameter was not necessary for that version?

Reply:
In old version of .NET (in this case .net 4.5.2), there was not built in json serializer. So it takes a dependency on an external package Newtonsoft.Json to do the serialization, which doesn't have an api to read the byteswritten_so_far, without which we cannot offer to honor the `MaxPayloadSizeInBytes`